documentation/docs/setup_a_node.mdx

```diff
@@ -60,16 +60,16 @@ apt-get install git npm build-essential cmake pkg-config librocksdb-dev llvm cla
 
 To run a Bee node in a macOS system, you will need to install the following packages using the [brew](https://brew.sh/) package manager:
 
-- [cmake](https://cmake.org/)
-- [npm](https://www.npmjs.com/)
+- [cmake](https://cmake.org/).
+- [npm](https://www.npmjs.com/).
 
 You can run the following command to install brew:
 
 ```shell
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 ```
 
-After the installer finishes, you can use brew to install the required packages by running:
+After the installer finishes, you can use the brew to install the required packages by running:
```

Comment:
I think it's either "brew" on its own, or "the brew package manager".

Reply:
👌